### DISCUSSION TITLE
Can't open ipynb with code server v4.1.0

### DISCUSSION THREAD

**Discussion by Zonghang-FU** [coder/code-server#5011] [Q&A]
Hello, I have test code server in CentOS, Ubuntu. But neither of them can open the ipynb. I use code-server: v4.1.0 with ms-toolsai.jupyter v2021.11.1001552333 and ms-python.python v2021.12.1559732655. I have try other version.But nothing change

<img width="870" alt="截屏2022-03-22 下午6 24 02" src="https://user-images.githubusercontent.com/56997567/159539667-f1b078bf-39df-46ab-a50f-8b8e7cc99af0.png">

**Comment by jsjoeio** [comment]
Are there any logs in the browser or on the server?

**Reply by Zonghang-FU** [reply]
Thanks for your help. It never works at http, but if I make a cert by mkcert and try to use https,ipynb works. It was the same at 4.20, 4.10

**Reply by jsjoeio** [reply]
Hmm...I think you need to use HTTPS or localhost then to make it work. I don't think that's something we can change since it uses service workers and they require a secure context.

**Reply by lg-nineoneninethreenine** [reply]
Wait, we could try the insecure origin flag on Chromium-based browsers, it will allow you to have HTTPS on any HTTP website.
